guides/v2.1/frontend-dev-guide/translations/theme_dictionary.md

```diff
@@ -6,28 +6,27 @@ functional_areas:
   - Theme
 ---
 
-## What's in this topic
-
-This topic describes how default strings can be changed in your custom {% glossarytooltip d2093e4a-2b71-48a3-99b7-b32af7158019 %}theme{% endglossarytooltip %} using [translation dictionaries]. It also gives a short overview of the order in which translations are searched and applied by the Magento application.
+Modify default strings in your custom {% glossarytooltip d2093e4a-2b71-48a3-99b7-b32af7158019 %}theme{% endglossarytooltip %} to load and use [translation dictionaries]. Learn more about locales, modifying strings, and how Magento searches and applies translations. 
```

Comment:
The last sentence here needs some thought.

Reply:
Removed it thanks!